exercises/space-age/src/example.cr

```diff
@@ -1,18 +1,36 @@
-module SpaceAge
-  extend self
+module Space
+  struct Age
+    SECONDS_PER_EARTH_YEAR = 31557600
+    def age_on_mercury
```

Comment:
Weren't these using a macro to build out the methods before? I thought that sparked the example change?

Reply:
Yes, but when you said

> Also, using macros is a great way to solve this (vs a hash)... however, these "examples" are basically just ways to test the test and generator. If people want examples, they should be looking at the ones in exercism.

I thought you meant to remove the macros from the example. I'd be glad to put them back.